<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="CentralWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background: transparent;
color: rgb(210, 210, 210);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frameMain">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(44, 49, 60);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frameSettings">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QFrame" name="mainSettings">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="1">
                <layout class="QVBoxLayout" name="labels_layout">
                 <item>
                  <widget class="QLabel" name="label_symbols">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background: transparent;
margin-left: 5px;</string>
                   </property>
                   <property name="text">
                    <string>Количество символов</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background: transparent;
margin-left: 5px;</string>
                   </property>
                   <property name="text">
                    <string>Прописные буквы</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background: transparent;
margin-left: 5px;</string>
                   </property>
                   <property name="text">
                    <string>Строчные буквы</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background: transparent;
margin-left: 5px;</string>
                   </property>
                   <property name="text">
                    <string>Цифры</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background: transparent;
margin-left: 5px;</string>
                   </property>
                   <property name="text">
                    <string> Спец. символы</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="buttons_layout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLineEdit" name="line_count">
                   <property name="focusPolicy">
                    <enum>Qt::StrongFocus</enum>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLineEdit {
    border: 2px solid gray;
   border-radius: 5px;
    padding: 0 8px;
    font: 10px;
    selection-background-color: darkgray;
}
</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0" colspan="2">
                <layout class="QVBoxLayout" name="generate_layout">
                 <item>
                  <widget class="QPushButton" name="btn_generate">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton {
    background-color: rgb(10, 10, 35);
    border: none;
    border-radius: 2px; /* Здесь вы можете настроить радиус закругления */
}
QPushButton:hover {
    background-color: rgb(15, 15, 50);
}
QPushButton:pressed {
    background-color: rgb(5, 5, 20);
}
</string>
                   </property>
                   <property name="text">
                    <string>Generate</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1" rowspan="2">
             <widget class="QFrame" name="frameList">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QListWidget" name="passwordsList">
                 <property name="styleSheet">
                  <string notr="true">QListWidget {
    border: none;
}
QListWidget::item {
    border-bottom: 1px solid lightgray;
    padding: 5px;
}
QListWidget::item:selected {
    background: lightblue;
    color: black;
}

QScrollBar:vertical, QScrollBar:horizontal {
    border: none;
    background: rgb(181,184,177);
    width: 5px; /* Для горизонтального скроллбара это будет высота */
    height: 5px; /* Только для горизонтального скроллбара */
    margin: 20px 0 20px 0;
    border-radius: 5px;
}

QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background: #a0a0a0;
    min-height: 20px; /* Для вертикального скроллбара */
    min-width: 20px;  /* Для горизонтального скроллбара */
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover, QScrollBar::handle:horizontal:hover {
    background: #909090;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical,
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    background: none;
    border: none;
    height: 0;
    width: 0; 
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical,
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}
</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QFrame" name="additionalSettings">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton {
    background-color: rgb(71,74,81);
    border: none;
    border-radius: 2px; /* Здесь вы можете настроить радиус закругления */
}
QPushButton:hover {
    background-color: rgb(15, 15, 50);
}
QPushButton:pressed {
    background-color: rgb(5, 5, 20);
}
</string>
                   </property>
                   <property name="text">
                    <string>Save</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background: transparent;
margin-left: 5px;</string>
                   </property>
                   <property name="text">
                    <string>Сохранить пароль</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
